### DISCUSSION TITLE
✅ 🔗 Ability to disable widgets / containers (Duplicate of #4246)

### DISCUSSION THREAD

**Discussion by mbaierl** [elementor/elementor#23984] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

To disable a section or widget on a page right now we have to use the responsive controls. By hiding it on desktop, tablet and mobile the element no longer shows up on the frontend.
However the element is still visible in the HTML source code.

Thjsi would be greatly improve performance when testing out new things or preparing landing pages.

### Describe the solution you'd like

Please add an option to disable an element completely, so it is also removed from the DOM.

### Describe alternatives you've considered

Responsive controls won't work as they leave the code in the DOM.

**Comment by nicholaszein** [comment]
Hello!

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.19.0, and improved with Elementor Pro 3.20.0! 🥳 

You can now use Display Conditions for widgets based on a variety of conditions, including date, time, user role, logged in/out, and others (more to come too). Check the [documentation for the new Display Conditions](https://elementor.com/help/show-and-hide-elements-on-a-page-display-conditions/) for widgets (Pro).

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
